### ISSUE TITLE
Contents of language attribute on HTML element missing in Techniques

### ISSUE THREAD

**Issue by hidde** [w3c/wcag#2005]
The `html` element on individual Technique pages in WCAG 2.1 reads:

```markup
<html xmlns="http://www.w3.org/1999/xhtml" xml:lang="" lang="">
``` 

There are no values in the `xml:lang`/`lang` attributes.

**Comment by hidde** [comment]
Moving this issue to [wcag](https://github.com/w3c/wcag) as it also exists in the live pages.

**Comment by giacomo-petri** [comment]
The issue appears to be resolved, so I'm closing it as fixed.

If you encounter this issue again please feel free to reopen it
